docs/kb/api-version-4.md

```diff
@@ -1,48 +1,98 @@
-# API Version 4
+# What is Version 4?
 
 In order to accommodate a new feature related to prospects,
```

Comment:
This should probably say in order to accommodate multiple prospects with the same email address.